### ISSUE TITLE
Any news about updated codes?

### ISSUE THREAD

**Issue by Cholowao** [fundon/vue-admin#264]

**Comment by ayoubfaouzi** [comment]
Is this even gonna happen one day ?

**Comment by fundon** [comment]
Sorry, I haven’t been very happy this year. This project has been suspended for a long time.

Do you need me to continue developing this project? @Cholowao @LordNoteworthy

**Comment by ayoubfaouzi** [comment]
Yes @fundon this admin dashboard has the most elegant UI I have seen for Vue.

Update the project so people can sponsor you, I will be happy to give something.

Cheers.

**Comment by fundon** [comment]
@LordNoteworthy Ok. Thanks for your support, I will update some details.

**Comment by aryalg** [comment]
What's going on about update? It's been a long time.

**Comment by fundon** [comment]
@aryalg I updated the [README](https://github.com/vue-bulma/vue-admin/blob/next/README.md).

**Comment by Tofandel** [comment]
Just discovered this and I have to say it looks awesome and I'd love to use it
So in the wait of the updated code, I'm giving you some encouragements :clap:
